easybuild/easyblocks/t/tau.py

```diff
@@ -237,32 +232,39 @@ def build_step(self):
         """No custom build procedure for TAU."""
         pass
 
+    def install_step(self):
+        """Create symlinks into arch-specific directories"""
+        super(EB_TAU, self).install_step()
+        # Link arch-specific directories into prefix
+        arch_dir = find_arch_dir(self.installdir)
+        self.log.debug('Found %s as architecture specific directory. Creating symlinks...', arch_dir)
+        for d in ('bin', 'lib'):
+            src = os.path.join(arch_dir, d)
+            dst = os.path.join(self.installdir, d)
+            if os.path.exists(dst):
```

Comment:
I changed it to `lexists` which also checks for broken symlinks. The message was changed to info instead of debug.

If it is added and is correct all is good, sanity check covers the case where it points/contains something wrong.